bindings/matrix-sdk-ffi/src/encryption.rs

```diff
@@ -413,16 +414,51 @@ impl Encryption {
 
     /// Get the E2EE identity of a user.
     ///
-    /// Returns Ok(None) if this user does not exist.
+    /// This method always tries to fetch the identity from the store, which we
+    /// only have if the user is tracked, meaning that we are both members
+    /// of the same encrypted room. Optionally it can request the user from the
+    /// homeserver if not found locally.
     ///
-    /// Returns an error if there was a problem contacting the crypto store, or
-    /// if our client is not logged in.
-    pub async fn get_user_identity(
+    /// # Arguments
+    ///
+    /// * `user_id` - The ID of the user that the identity belongs to.
+    ///
+    /// * `request_from_homeserver_if_needed` - If true, the identity will be
+    ///   requested from the homeserver if it is not found in the crypto store.
+    ///   When `true` the identity returned is guaranteed to be up-to-date.
+    ///
+    /// Returns a `UserIdentity` if one is found. Returns an error if there
+    /// was an issue with the crypto store or with the request to the
+    /// homeserver.
+    ///
+    /// This will always return `None` if the client hasn't been logged in.
+    pub async fn user_identity(
         &self,
         user_id: String,
+        request_from_homeserver_if_needed: bool,
```

Comment:
I am wondering in which case(s) an sdk user could set the parameter `request_from_homeserver_if_needed` to `false`.